VoxelSniperCore/src/main/java/com/github/kevindagame/voxelsniper/events/player/PlayerSnipeEvent.java

```diff
@@ -2,31 +2,39 @@
 
 import com.github.kevindagame.brush.IBrush;
 import com.github.kevindagame.voxelsniper.entity.player.IPlayer;
+import com.github.kevindagame.voxelsniper.events.Cancellable;
 import com.github.kevindagame.voxelsniper.events.EventPriority;
 import com.github.kevindagame.voxelsniper.events.HandlerList;
 
+import java.util.List;
 import java.util.function.Consumer;
 
+import com.github.kevindagame.voxelsniper.location.VoxelLocation;
 import org.jetbrains.annotations.NotNull;
 
-public class PlayerSnipeEvent extends PlayerEvent<PlayerSnipeEvent> {
+public class PlayerSnipeEvent extends PlayerEvent<PlayerSnipeEvent>implements Cancellable {
 
     private static final HandlerList<PlayerSnipeEvent> handlers = new HandlerList<>();
     private final IBrush brush;
-    private final boolean success;
-
-    public PlayerSnipeEvent(IPlayer p, IBrush brush, boolean success) {
+    private final List<VoxelLocation> positions;
+    private EventResult status;
+    public PlayerSnipeEvent(final IPlayer p, final IBrush brush, List<VoxelLocation> positions) {
         super(p);
         this.brush = brush;
-        this.success = success;
+        this.status = EventResult.DEFAULT;
+        this.positions = positions;
     }
 
     public IBrush getBrush() {
         return brush;
     }
 
-    public boolean isSuccessful() {
-        return success;
+    public List<VoxelLocation> getPositions() {
+        return positions;
+    }
```

Comment:
Little bit of documentation might be nice here as well, could someone listening to this event change the list of positions? Could I add additional positions?

Reply:
Will add this